src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,62 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String readString = readFile(fromFileName);
+        String report = createReport(readString);
+        writeFile(toFileName, report);
+    }
+
+    private String readFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        File readFile = new File(fromFileName);
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(readFile))) {
+            String localString = bufferedReader.readLine();
+            while (!(localString == null)) {
```

Comment:
```suggestion
            String localString;
            while ((localString = bufferedReader.readLine()) != null) {
```
and delete line 24 and all will work properly

Reply:
I did it.

Reply:
And I had to deleted 
localString = bufferedReader.readLine();
for code be worked.